rules/Privatization/TypeManipulator/TypeNormalizer.php

```diff
@@ -27,7 +27,7 @@
     /**
      * @var int
      */
-    private const MAX_PRINTED_UNION_DOC_LENGHT = 60;
+    private const MAX_PRINTED_UNION_DOC_LENGHT = 77;
```

Comment:
@TomasVotruba I updated to use `77` as max length union to show the fix, otherwise, it just `\Iterator<mixed>` for this fixture test

https://github.com/rectorphp/rector-src/pull/7298/files#diff-2a2097385262575254922102ea5d5304cc01549cfcbdc51ef87c240199f2c4d5

The fix for this case is loop when inner is `ConstantArrayType` too.

https://github.com/rectorphp/rector-src/pull/7298/files#diff-a5bab2fbc05ee30b694214b4e5b5c805297d9e916fc482845f92e70fd9bd3c06

Proven works at 

https://phpstan.org/r/3a7c2934-6256-48c6-a4ba-351f42d4d0df

Reply:
This might be tricky to keep in readable line. The longer the docblock will be, the more mess might get int. But let's give 77 a try.